packages/did-core-test-server/suites/did-url-dereferencing/did-url-dereferencing.js

```diff
@@ -22,38 +14,45 @@ const didUrlDereferencingTests = (execution, expectedOutcome) => {
       });
 
       it.todo('3.2.2 Relative DID URLs - When resolving a relative DID URL reference, the algorithm specified in RFC3986 Section 5: Reference Resolution MUST be used.');
-      it('A conformant DID URL as a single string.', async() => {
+      it('7.2 DID URL Dereferencing - A conformant DID URL as a single string.', async() => {
         if (! dereferencingMetadata.hasOwnProperty('error') || dereferencingMetadata['error'] !== 'invalidDidUrl') {
-          expectConformantDidUrl(didUrl);
+          expect(didUrl).toBeValidDidUrl();
         }
       });
-
-      it.todo('To dereference a DID fragment, the complete DID URL including the DID fragment MUST be used.');
-
-      it('This input is REQUIRED.', async () => {
+      it('7.2 DID URL Dereferencing - To dereference a DID fragment, the complete DID URL including the DID fragment MUST be used.', async () => {
+        if(didUrl.includes('#')) {
+          const didUrlWithoutFragment = didUrl.substring(0, didUrl.indexOf('#'));
+          const executionWithoutFragment = utils.findExecutionByDidUrl(implementation, didUrlWithoutFragment);
+          if (executionWithoutFragment !== undefined) {
+            const contentStreamWithoutFragment = executionWithoutFragment.output.contentStream;
+            expect(contentStreamWithoutFragment).not.toBe(contentStream);
+          }
+        }
+      });
+      it('7.2 DID URL Dereferencing - This input is REQUIRED.', async () => {
         expect(didUrl).not.toBeFalsy();
       });
     });
-    describe('dereferenceOptions', () => {
-      it('A metadata structure.', async () => {
-        expectConformantMetadataStructure(dereferenceOptions);
+    describe('dereferencingOptions', () => {
+      it('7.2 DID URL Dereferencing - A metadata structure.', async () => {
+        utils.expectConformantMetadataStructure(dereferenceOptions);
```

Comment:
This can be merged with my part of the test.

Reply:
I agree, I was planning to do this later (i.e. better re-use existing test code), after this PR is merged.